server/src/main/java/org/elasticsearch/indices/ShardLimitValidator.java

```diff
@@ -197,21 +198,16 @@ private Optional<String> checkShardLimit(int newShards, int newFrozenShards, Clu
      * "cluster.max_shards_per_node" setting if the shards are going on normal nodes. This check does not guarantee that the number of
      * shards can be added, just that there is theoretically room to add them without exceeding the shards per node configuration.
      * @param numberOfNewShards The number of primary shards that we want to be able to add to the cluster
-     * @param replicas The number of replcas of the primary shards that we want to be able to add to the cluster
-     * @param state The cluster state, used to get cluster settings and to get the number of open shards already in the cluster
-     * @param frozenNodes If true, check whether there is room to put these shards onto frozen nodes. If false, check whether there is room
-     *                    to put these shards onto normal nodes.
+     * @param replicas          The number of replicas of the primary shards that we want to be able to add to the cluster
```

Comment:
The description in these javadocs still talks about frozen nodes. We need to pull that out if it's no longer an option to deal with them.